benchmarks/src/test/scala/pl/touk/nussknacker/engine/benchmarks/interpreter/ManyParamsInterpreterBenchmark.scala

```diff
@@ -8,7 +8,7 @@ import pl.touk.nussknacker.engine.api.{Context, MethodToInvoke, ParamName, Servi
 import pl.touk.nussknacker.engine.build.ScenarioBuilder
 import pl.touk.nussknacker.engine.canonicalgraph.CanonicalProcess
 import pl.touk.nussknacker.engine.graph.expression.Expression
-import pl.touk.nussknacker.engine.spel.Implicits._
```

Comment:
I can't see removal of Implicits object. Do you removed it? If not, please remove the dead code. Also, please add a migration guide entry.